docs/job/push-to-docker.md

```diff
@@ -1,11 +1,14 @@
 # push-to-docker
 
 This job builds a docker image and pushes it to a registry.
-It requires the build context and Dockefile to be present at the root of worksapce directory.
+It uses the Dockerfile at the root of the workspace directory and the root directory as build context by default.
+Otherwise, it is possible to specify the used Dockerfile and build context by specifying `dockerfile` and `build-context` arguments.
```

Comment:
```suggestion
Otherwise, it is possible to specify the Dockerfile and build context to use with `dockerfile` and `build-context` arguments respectively.
```